src/pages/[platform]/build-a-backend/storage/list/index.mdx

```diff
@@ -129,7 +129,7 @@ To list other user's objects:
 import { list } from 'aws-amplify/storage';
 
 try {
-  const response = list({ prefix: 'photos/', options:  {
+  const response = await list({ prefix: 'photos/', options:  {
```

Comment:
on this page, lots of the code formatting is also slightly off. Do you mind fixing all those as well as you're at it?

Reply:
for example `prefix` on a new line etc.

Reply:
sounds good